crates/turborepo-paths/src/relative_unix_path_buf.rs

```diff
@@ -42,7 +42,11 @@ impl RelativeUnixPathBuf {
             .filter(|(_, byte)| **byte == b'\"' || **byte == b'\n')
         {
             writer.write_all(&self.0[i..to_escaped_index])?;
-            writer.write_all(&[b'\\', *byte])?;
+            if *byte == b'\n' {
+                writer.write_all(&[b'\\', b'n'])?;
```

Comment:
This is a bug fix, but now that we're using `libgit2`, I don't think we need this code at all

Reply:
Removed this code.